src/js/protocols/stm32usbdfu.js

```diff
@@ -1097,6 +1100,7 @@ STM32DFU_protocol.prototype.leave = function () {
 
 STM32DFU_protocol.prototype.cleanup = function () {
     const self = this;
+    GUI.log('running the cleanup function');
 
     self.releaseInterface(0);
```

Comment:
I think that running the release interface function may fail when `cleanup` is being run in response to an error somewhere.